modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data with rules
+You can configure the `FlowCollector` to filter eBPF flows using rules as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A few sample configurations are as follows:
++
+
+[%collapsible]
+.Filter Kubernetes service traffic to specific POD IP endpoint
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        action: Accept  <1>
+        cidr: 172.210.150.1/24 <2>
+        protocol: SCTP
+        direction: Ingress
+        destPortRange: 80-100
+        peerIP: 10.10.10.10
+----
+<1> The required `action` parameter describes the action to be taken for the flow filter rule. Possible values are `Accept` and `Reject`. 
+<2> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule, and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6, respectively.
+====
++
+[%collapsible]
+.See flows to any addresses outside the cluster 
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        action: Accept  
+        cidr: 0.0.0.0/0 <1>
+        protocol: TCP
+        direction: Egress
+        source_port: 100
+        peerIP: 192.168.127.12 <2>
+----
+
+<1> The `cidr` value of `0.0.0.0/0` matches against any OP address.
```

Comment:
what is `OP address` ?

Reply:
Ah, that's a typo  it should be IP

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/74622/files#diff-ced28301ee53effb3bbb412410346997b450c48937bcd8da10eec15ef14882b1R72